src/Utils/BeanDescriptor.php

```diff
@@ -465,7 +466,7 @@ private function getPivotTableDescriptors(): array
      */
     private function checkForDuplicate(MethodDescriptorInterface $descriptor): void
     {
-        $name = $descriptor->getName();
+        $name = mb_strtolower($descriptor->getName());
```

Comment:
I think we can safely use `strtolower`.
We are dealing with method names here and PHP internals is case sensitive in the "old fashioned" way. I believe PHP internals does not handle UTF8 so it is internally putting method names in lower case using `strtolower`.

Reply:
I just tested that:
```php
function é() {}
function É() {}
```
No error, you are perfectly right, I'll change that to `strtolower`!